cachyos-gaming-meta/.SRCINFO

```diff
@@ -1,64 +1,30 @@
 pkgbase = cachyos-gaming-meta
 	pkgdesc = Meta package for Gaming dependencies
 	pkgver = 1.0.0
-	pkgrel = 1
+	pkgrel = 2
 	epoch = 1
 	arch = any
-	depends = alsa-lib
 	depends = alsa-plugins
-	depends = dosbox
 	depends = giflib
-	depends = glfw
```

Comment:
Does anyone know if this is necessary for any games? AFAICT this is just a dependency for `mangohud`

Reply:
This dependency may be needed for Minecraft if you want to use a system library, such as for native Wayland support on 1.13+.

Reply:
But I guess it's okay to remove it from depends since mangohud requires it too.

Reply:
> This dependency may be needed for Minecraft if you want to use a system library, such as for native Wayland support on 1.13+.

I'll add it back then, considering `mangohud` is not part of this meta package